hbase-backup/src/test/java/org/apache/hadoop/hbase/backup/TestIncrementalBackup.java

```diff
@@ -79,6 +95,7 @@ public TestIncrementalBackup(Boolean b) {
 
   // implement all test cases in 1 test since incremental
   // backup/restore has dependencies
+  @SuppressWarnings("checkstyle:MethodLength")
```

Comment:
Let me know if there's a better way to go about this. We could split the test up, but I think it's nice to have one single test for the entire backup/restore process. It helps simulate real-life scenarios where we're constantly taking backups of the same table.